benchmark/buffers/buffer-hex-decoded.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const common = require('../common.js');
+
+const bench = common.createBenchmark(main, {
+  len: [64, 1024],
+  n: [1e6],
+});
+
+function main({ len, n }) {
+  const buf = Buffer.alloc(len);
+
+  for (let i = 0; i < buf.length; i++)
+    buf[i] = i & 0xff;
+
+  const plain = buf;
+
+  bench.start();
+
+  for (let i = 0; i < n; i += 1)
+    plain.toString('hex');
```

Comment:
Assign the result of the toString to a variable and then `assert.ok` to avoid v8 deadcode elimination